src/ingestion-server/server/images/nginx/config/nginx.conf

```diff
@@ -40,10 +40,12 @@ http {
             # limit_req_status 444;
 
             add_header Content-Type text/plain;
-            add_header 'Access-Control-Allow-Origin' '%%SERVER_CORS_ORIGIN%%';
-            add_header 'Access-Control-Allow-Credentials' 'true';
-            add_header 'Access-Control-Allow-Methods' 'GET, POST, OPTIONS';
-            add_header 'Access-Control-Allow-Headers' 'DNT,X-CustomHeader,Keep-Alive,User-Agent,X-Requested-With,If-Modified-Since,Cache-Control,Content-Type';
+            if ($http_origin ~* (%%SERVER_CORS_ORIGIN%%)) {
```

Comment:
Will it work if the `SERVER_CORS_ORIGIN` is blank or empty?

Reply:
How about inputting `http://localhost:8080`? Will only allow the requests from localhost with port 8080?

Reply:
Good point,  
After refactoring, if SERVER_CORS_ORIGIN is blank or empty, no domain will be allowed .

For localhost, if input http://localhost:8080, will only allow http://localhost:8080/xxx, other port will not allow.